analyze-project/action.yml

```diff
@@ -72,7 +73,9 @@ runs:
         with open(os.environ["GITHUB_OUTPUT"], "a") as output:
             print(f"mypy={is_installed('mypy')}", file=output)
             print(f"pyright={is_installed('pyright')}", file=output)
-      shell: python
+        EOF
+      working-directory: ${{ inputs.project-directory }}
```

Comment:
question, did you try just setting the working dir (without switching to running a bash script)?

Reply:
I guess if we did that, we would also need to set the "shell" to be poetry's install python (in some way)...